darwin/importer/importer.py

```diff
@@ -555,44 +560,66 @@ def _import_annotations(
     dataset: "RemoteDataset",
     append: bool,
     delete_for_empty: bool,
-) -> None:
+) -> Tuple[dt.ErrorList, dt.Success]:
+    errors: dt.ErrorList = []
+    success: dt.Success = dt.Success.SUCCESS
+
     serialized_annotations = []
     for annotation in annotations:
         annotation_class = annotation.annotation_class
         annotation_type = annotation_class.annotation_internal_type or annotation_class.annotation_type
-        annotation_class_id = remote_classes[annotation_type][annotation_class.name]
-
-        if isinstance(annotation, dt.VideoAnnotation):
-            data = annotation.get_data(
-                only_keyframes=True,
-                post_processing=lambda annotation, data: _handle_subs(
-                    annotation, _handle_complex_polygon(annotation, data), annotation_class_id, attributes
-                ),
+
+        try:
```

Comment:
I know this paradigm is kind of pythonic with EAFP but I'm not a huge fan of embedded try/except statements. Not blocking by any means, but I prefer 
```
if key in remote_classes[annotation_type]
...
elif key.strip() in remote_classes[annotation_type]
```
Thoughts?